lib_eio_linux/eio_linux.ml

```diff
@@ -1017,21 +1021,21 @@ let rec run ?(queue_depth=64) ?(block_size=4096) main =
               fn st k;
               schedule st
             )
-          | ERead args -> Some (fun k -> 
+          | ERead args -> Some (fun k ->
               let k = { Suspended.k; fibre } in
               enqueue_read st k args;
               schedule st)
-          | Close fd -> Some (fun k -> 
+          | Close fd -> Some (fun k ->
```

Comment:
It seems my nvim settings has automatically removed trailing newlines and made the diff large. Hope this is okay.

Reply:
Sure. You added them in the first place ;-)